docs/use-cases/portfolio-management/README.md

```diff
@@ -0,0 +1,44 @@
+---
+sidebar_navigation:
+  title: OpenProject Use-Cases: Portfolio Management and Custom Reporting
```

Comment:
Please use only short text for the title of entry in the sidebar, so the entries don't span in multiple lines.
e.g. "Portfolio Management and Custom Reporting" is enough.